app/Mail/AnonymousEmail.php

```diff
@@ -14,20 +14,14 @@ class AnonymousEmail extends Mailable
 
     public $committee;
 
-    public $message_content;
-
-    public $hash;
-
     /**
      * Create a new message instance.
      *
      * @return void
      */
-    public function __construct(Committee $committee, $message_content, $message_hash)
+    public function __construct(Committee $committee, public $message_content, public $hash)
```

Comment:
did not know that this is a thing in php :confetti_ball:

Reply:
That is why this package will help us, it will run on every PR just like our linter and therefore make sure the code is up to date with the newest php features once we toggle them on.